universal-application-tool-0.0.1/app/services/applicant/ApplicantQuestion.java

```diff
@@ -248,7 +238,12 @@ public TextQuestion() {
 
     @Override
     public boolean hasErrors() {
-      // There are no inherent requirements in a text question.
+      if (hasValue()) {
+        return getTextValue()
+            .filter(s -> s.length() >= getQuestionDefinition().getMinLength())
```

Comment:
issue (blocking): I think we need to avoid checking against these validations if they don't exist (Maybe I am just missing something though 😃 ).

Reply:
Do you mean `getMinLength` should return an optional?

Reply:
Yep exactly - the admin may not configure any min or max.

Reply:
Sounds good, thanks for the clarification!

Reply:
Done.